libraries/SITL/SITL.cpp

```diff
@@ -65,8 +65,14 @@ SIM *SIM::_singleton = nullptr;
 
 // table of user settable parameters
 const AP_Param::GroupInfo SIM::var_info[] = {
-    
+
+    // @Param: DRIFT_SPEED
+    // @DisplayName: Gyro drift speed
+    // @Description: Gyro drift rate of change in degrees/second/minute
     AP_GROUPINFO("DRIFT_SPEED",    5, SIM,  drift_speed, 0.05f),
+    // @Param: DRIFT_TIME
+    // @DisplayName: Gyro drift time
+    // @Description: Gyro drift duration of one full drift cycle (period in minutes)
```

Comment:
Is this really in minutes?

Reply:
Appears to be: `AP_Float drift_time;  // period in minutes`